### PR TITLE
Org rename

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,11 +20,11 @@ do](https://18f.gsa.gov/2017/01/04/tracking-the-us-governments-progress-on-movin
 
 `pshtt` is a collaboration between the Department of Homeland
```
Comment (h-m-f-t) [inline README.md]: I think that in most instances we can replace "Department of Homeland Security" with CISA (expanded or no). Yes, CISA is part of DHS, but no one says DHS TSA or DHS CBP. We're just CISA.

File: setup.py [hunk 2]
```diff
@@ -28,10 +28,10 @@ def readme():
     # NCATS "homepage"
     url='https://www.dhs.gov/cyber-incident-response',
```
Comment (h-m-f-t) [inline setup.py]: It's not great, but this page is probably the most representative page we have going for us right now:

https://www.us-cert.gov/resources/ncats

**Review by h-m-f-t — COMMENTED** [review]
Small suggestions, but otherwise 👍

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: Fixed in 3f30e07.

File: setup.py (hunk 2 above)
Comment (jsf9k) [reply setup.py]: Fixed in e2c34d3.

**Review by h-m-f-t — APPROVED** [review]

**Pull request merged**